theme.js

```diff
@@ -41,50 +50,39 @@ define(function(require, exports, module) {
         // when plugin is loaded
         plugin.on("load", function() {
 
-            // create button
-            button = new imports.ui.button({
-                "command": "toggleTheme",
-                "skin": "c9-menu-btn",
-                "tooltip": "Theme",
-                "visible": true
+            menuItem = new ui.item({
+                type: "check",
+                caption: "ToggleTheme",
+                onclick: toggleTheme
             });
 
+            menus.addItemByPath(
+                "View/PresentationDiv", new ui.divider(), 3, plugin
```

Comment:
Looking at it, I'm not sure we even need it because we're not making a new divider, so I'm taking it out.